src/lang/fr.json

```diff
@@ -250,6 +250,7 @@
   "Naturerlebnispark": "Parc naturel périurbain",
   "Nein": "Non",
   "Nicht barrierefreier Bahnhof": "Gare non accessible",
+  "nicht treppenfrei": "Pas sans escalier",
```

Comment:
@edellucien is that fench ? :)

Reply:
Litteral translation of 'nicht treppenfrei' or 'not without stairs' indeed.
But even in other language it seems weird.

Reply:
`Presence d'escaliers`  ou `avec escaliers` ou `Escaliers existant`

Reply:
tu manies la prose comme personne :smile: